freqtrade/exchange/exchange.py

```diff
@@ -1406,10 +1549,14 @@ async def _async_fetch_trades(self, pair: str,
         except ccxt.BaseError as e:
             raise OperationalException(f'Could not fetch trade data. Msg: {e}') from e
 
+    def batch_size(self) -> int:
+        return 1000
```

Comment:
Is this a constant (per exchange) that defines how many trades we expect per trade?

If yes, then please move it to `_ft_has` (per exchange?).
`ohlcv_candle_limit` does basically the same but for candle data.